curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/6464ab8c06ea92f30bc548d5.md

```diff
@@ -26,7 +26,7 @@ assert.property(
 )
 ```
 
-You should set the value of the `msg` property to `decimalToBinary(5) returns "10" + 1 (5 % 2). Then it pops off the stack.`
+You should set the value of the `msg` property to `'decimalToBinary(5) returns "10" + 1 (5 % 2). Then it pops off the stack.'`
```

Comment:
You have to change all single quotes like this to double quotes.

https://github.com/freeCodeCamp/freeCodeCamp/issues/51036#issuecomment-2158103315

Reply:
Hi @Ksound22, does this mean I have to change the inner double quotes to single quotes?

Reply:
Instead of this:
`'decimalToBinary(5) returns "10" + 1 (5 % 2). Then it pops off the stack.'`

You should have this:
`"decimalToBinary(5) returns "10" + 1 (5 % 2). Then it pops off the stack."`

Reply:
@Ksound22 no problems with double quotes within double quotes, right?

Reply:
i think this will cause more of a confusion to the camper as there is a double quotes inside of the string.
we would either have to use backticks or expect the camper to know to use backticks in this case to escape the double quotes in the string. 
by using backticks we are letting the camper to just copy and paste the answer, as the task clearly asks the to use given string as the value of property`msg`

Reply:
Addressed following @RGHANILOO suggestion